swift/json_object_array.swift

```diff
@@ -0,0 +1,21 @@
+//Script writes an array of objects to stdout
+import Foundation
+
+guard CommandLine.arguments.count > 1 else {
+    print("Usage: swift script.swift <arg1> [<arg2> ...]")
+    exit(1)
+}
+
+let args = Array(CommandLine.arguments.dropFirst())
```

Comment:
```suggestion
let args = CommandLine.arguments.dropFirst()
```

Let's remove the `Array(...)` wrapper from the other swift scripts, too

Reply:
Make sure to get all the other instances of `Array(...)`